### ISSUE TITLE
Bad imports are only a warning in the UI codebase

### ISSUE THREAD

**Issue by petong** [AugurProject/augur#423]
My crap code was able to merge into the codebase which included a bad import statement as it is currently only a warning:

```WARNING in ./src/modules/escape-hatch/actions/withdraw-funds-in-emergency.js
52:21-53 "export 'updateDisputeCrowdsourcerBalance' was not found in 'modules/markets/actions/update-markets-data'```

This sort of thing should probably be a full blown error that prevents merging.
...

@justinbarry can we force warnings as errors?
 -bthaile
...

We need to figure out how to set this for both eslint and webpack.
 -justinbarry
...

We now build the client in CI to prevent this sort of thing from getting into master. See ch7893 
 -justinbarry
...

@justinbarry Can you please let us know if there's more work to do on this? If not, can you please close it?
 -adrake33
...

Ha! Looks like this is held up by rewire. We are working on replacing it. See ch15830 
 -justinbarry

**Comment by justinbarry** [comment]
@petong As long as we are building the ui during the CI process these issues will be identified by webpack.

I'll close.
